core/vm/jump_table.go

```diff
@@ -216,160 +251,183 @@ func newFrontierInstructionSet() JumpTable {
 			maxStack:    maxStack(0, 0),
 			halts:       true,
 			valid:       true,
+			internal:    true,
```

Comment:
do you know why this new field internal are all true?   if it's all true, what's the point of having this field?

Reply:
I can see ~10 instructions that are not market as internal=true, for example: EXTCODEHASH, CREATE2, BALANCE, EXTCODESIZE, EXTCODECOPY, BLOCKHASH, COINBASE, ... Those are not allowed to be used in the AA transaction code before PAYGAS is executed.